backend/src/main/resources/application.yaml

```diff
@@ -43,4 +43,9 @@ server:
 #    key-store-password: password
 #    key-store-type: pkcs12
 #    key-alias: localhost
-#    key-password: password
\ No newline at end of file
+#    key-password: password
+#logging:
+#  level:
+#    KC_WEB_CLIENT: DEBUG
+#    org.apache.coyote.http11.Http11InputBuffer: DEBUG
+#    root: DEBUG
```

Comment:
I didn't mean to add this, but it's a nice example of how to configure the logger. KC_WEB_CLIENT is the client that sends requests to Keycloak, and Http11InputBuffer is a class that logs the user's token.